lib/fathead/numpy/parse.py

```diff
@@ -206,16 +213,52 @@ def get_abstract(section_div):
         rendered_returns = RETURNS_TEMPLATE.format(returns=returns)
     else:
         rendered_returns = ""
+    
+    examples = get_examples(section_div, "rubric")
+    if examples:
+        rendered_examples = EXAMPLES_TEMPLATE.format(examples=examples)
+    else:
+        rendered_examples = ""    
 
     abstract = ABSTRACT_TEMPLATE.format(
         rendered_information=scrub_text(rendered_information),
         rendered_signature=scrub_text(rendered_signature),
         rendered_parameters=scrub_text(rendered_parameters),
-        rendered_returns=scrub_text(rendered_returns)
+        rendered_returns=scrub_text(rendered_returns),
+        rendered_examples=scrub_text(rendered_examples)
     )
     return abstract
 
 
+def get_examples(section_div, examples_class):
+    """Parse and return the parameters or returns of the documentation topic.
+
+    Parameters
+    ----------
+    section_div : bs4.BeautifulSoup
+        The BeautifulSoup object corresponding to the div with the "class"
+        attribute equal to "section" in the html doc file.
+
+    examples_class: Str
+        The value of the "class" attribute of the <p> tag within section_div.
+        If "rubric" is present, then only the examples for the topic is there.
+
+    Returns
+    -------
+    Str:
+        The examples for the topic.
+    """
+    examples_present = section_div.find("p", attrs={"class": examples_class})
+    if not examples_present:
+        return
+    examples = section_div.find("pre")
```

Comment:
This doesn't look right. You're extracting the first "pre" tag from `section_div` instead of `examples_present`. The first `pre` tag from `section_div` might not necessarily be the the first example (though I suspect in many cases it will be).